source/form.html

```diff
@@ -106,45 +281,109 @@ <h1 class="page-main__title page-main__title--form">Подбор програм
+                  fill="currentColor" />
+              </g>
+              <defs>
+                <clipPath id="clip0">
+                  <rect width="20" height="20.0688" fill="currentColor" transform="translate(0 0.320435)" />
+                </clipPath>
+              </defs>
+            </svg>
+            <label class="form__label form__label--contacts form__label--contacts--phone" for="owner-phone">Телефон:*
+            </label>
+          </div>
+        </div>
       </div>
-
       <div class="form__comments">
-        <h2 class="form__subtitle">Комментарий</h2>
+        <h2 class="form__subtitle form__subtitle--comments">Комментарий</h2>
         <label class="visually-hidden" for="comments-textarea">Повадки кота.</label>
         <textarea class="form__input form__input--textarea" id="comments-textarea"
           placeholder="Расскажите обо всех повадках кота"></textarea>
       </div>
 
       <div class="form__other-products">
         <h2 class="form__subtitle form__subtitle--other-products">Дополнительно</h2>
-        <label class="form__label checkbox-label">Сахарозаменитель<input
-            class="form__input form__input--active checkbox" type="checkbox" checked></label>
-        <label class="form__label checkbox-label">Питьевая вода<input class="form__input checkbox"
-            type="checkbox"></label>
-        <label class="form__label checkbox-label">Молоко<input class="form__input checkbox" type="checkbox"></label>
-        <label class="form__label checkbox-label">Витамины<input class="form__input checkbox" type="checkbox"></label>
+        <div class="form__checkbox-list">
+          <div class="form__fieldset form__fieldset--checkbox">
+            <input class="form__input checkbox__input" type="checkbox" id="ckeckbox-sugar" checked>
```

Comment:
```suggestion
            <input class="form__input checkbox__input" type="checkbox" id="ckeCkbox-sugar" checked>
```

поправь русскую букву на латинскую